src/WinRT.Runtime/Interop/IActivationFactory.cs

```diff
@@ -18,6 +21,14 @@ namespace WinRT.Interop
 #endif
     interface IActivationFactory
     {
+        /// <summary>
+        /// Activates an instance of the target WinRT type.
+        /// </summary>
+        /// <returns>The resulting instance.</returns>
+        /// <exception cref="NotSupportedException">
+        /// Thrown if the operation is not supported by the activation factory type. For instance,
+        /// that is the case is the associated type is static or does not have a default constructor.
```

Comment:
```suggestion
        /// that is the case if the associated type is static or does not have a default constructor.
```

Reply:
Fixed in 9ce3b6c601b6da7155ee5a1354b6f51841f6b99e.